perun/utils/view_helpers.py

```diff
@@ -0,0 +1,206 @@
+"""Collection of helper functions for working with bokeh graphs"""
+from __future__ import annotations
```

Comment:
Can you elaborate, why you require this import?

Reply:
Especially in older Python versions, this import is needed to enable certain typing syntax sugar (e.g., using | instead of Union, or direct references of types in type aliases and type variables, ...). In this particular file, however, this import is needed so that pylint does not report false alarms regarding typing syntax for 3.8.

Reply:
That's neat, I will use it as well, since I wanted to use `|` as well.

Maybe consider adding comment `# this imports new typing annotations such as | operator for Union of types` or something like this.

Reply:
Although I can see the usefulness of such comment, it would quickly become tedious to maintain, I think. In my experience with typed codebases, this import is very common when using some of the more recent typing features and the comments would quickly fall out of date when bumping the minimal supported Python version of Perun.

Reply:
Yes, I wasn't so sure with it as well. I also consider some extremely short comment on the precise line, such as `from __future__ import annotation # import typing for backward compatibility`, but I guess it will suffice. I myself now know about this trick, and future contributors will surely learn it as well. Resolved.